docs/team/yuchenlichuck.md

```diff
@@ -1,24 +1,182 @@
-  
+
+## PROJECT: [CAFS](https://github.com/AY1920S2-CS2113-T14-3/tp/)
+
+
+
+## Overview
+
+* [1. CAFS Introduction](#introduction)
+
+* [2. Summary of Contributions](#summary-contribution)
++ [2.1 Code contribution](#code-contribution)
+  + [2.2 Enhancements implemented](#enhancement-implemented)
+  + [2.3 Contributions to documentation](#contribute-documentation)
+  + [2.4 Contributions to the Developer Guide](#contribute-developerguide)
+  + [2.5 Contributions to team-based tasks](#contribute-team)
+  + [2.6 Review/Mentoring contributions](#review-mentoring)
+  + [2.7 Contributions beyond the project team](#contribute-team)
   
-## PROJECT: CAFS
-Va CLI calender-like task scheduler that supports task and class schedule adding. It is simple to use, and comes with a save function to remember your tasks. The user interacts with the application using a CLI. The project is mainly written in Java.
+* [3. Contributions to the User Guide](#contribute-ug)
 
+* [4. Contributions to the Developer Guide](contribute-dg)
 
+<a name="introduction"></a>
 
+### CAFS Introduction 
+
+Va CLI calender-like task scheduler that supports task and class schedule adding. It is simple to use, and comes with a save function to remember your tasks. The user interacts with the application using a CLI. The project is mainly written in Java.
+
+<a name="summary-contribution"></a>
 
 ### Summary of contributions
+
+<a name="code-contribution"></a>
+
+#### Code contribution
+
+You can view my code contribution for this project: [HERE](https://nus-cs2113-ay1920s2.github.io/tp-dashboard/#search=yuchenlichuck&sort=groupTitle&sortWithin=title&since=2020-03-01&timeframe=commit&mergegroup=false&groupSelect=groupByRepos&breakdown=false)
+
 - Test Code: AddCommandTest, test if adding is right or out of boundary.
 - Refactor: Extract different method's constants into messages class.
-- Time and Date Format: Change the time and date format, so that the tool can recognise whether the date or time wrong or right (With Leap year).
\ No newline at end of file
+- Time and Date Format: Change the time and date format, so that the tool can recognise whether the date or time wrong or right (With Leap year).
+
+<a name="enhancement-implemented"></a>
+#### Enhancements implemented
+
+
+<a name="contribute-documentation"></a>
+#### Contributions to documentations:
+
+<a name="contribute-developerguide"></a>
+#### Contributions to the DG
+
+#### Contributions to team-based tasks:
+
+<a name="review-mentoring"></a>
+
+#### Review/Mentoring contributions: 
+
+<a name="contribute-team"></a>
+
+#### Contributions beyond the project team:
+
+<a name="contribute-ug"></a>
+
+### Contributions to the User Guide (Extracts)
+
+
+
+### List
+
+#### **List** 
+
+List all tasks
+
+* **Usage**: `list`
+
+#### **List Category**
+
+List tasks belong to a specific category
+
```

Comment:
Instead of specifying the usage here, maybe you can consider to delete this part and update in UG. Currently, the ppp has many competitive lines, what we write in ppp is just our task but not the specification of the task. The specification of the command should be put in UG.

Reply:
Well, actually, I mean the leap year. I write it in the last version when we use the data format that couldn't recognize leap year. But I see you now change it, so I may delete it later

Reply:
thanks, i would delete later

Reply:
I would suggest to first change the ppp and the UG then merge this branch.